backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/controller/WindowRestController.java

```diff
@@ -781,6 +786,24 @@ private DocumentZoomIntoInfo getDocumentFieldZoomInto(
 			final String tableName = adTableDAO.retrieveTableName(adTableId);
 			return DocumentZoomIntoInfo.of(tableName, recordId);
 		}
+		// label field
+		else if (field.getDescriptor().getWidgetType() == DocumentFieldWidgetType.Labels)
+		{
+			final LabelsLookup lookup = LabelsLookup.cast(field.getDescriptor().getLookupDescriptor().orElse(null));
```

Comment:
is the lookup-descriptor always present because the field is a label? if yes i suggest

```suggestion
			final LabelsLookup lookup = LabelsLookup.cast(field.getDescriptor().getLookupDescriptor().orElseThrow(() -> new AdempiereException("Because the widget type is Labels, expect a LookupDescriptor").setParameter("field", field)));
```